src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,18 @@
 import React from 'react';
+import { Comment } from '../../types/Comments';
+import { CommentInfo } from '../CommentInfo/CommentInfo';
+import './CommentList.scss';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  comments: Comment[];
+};
+
+export const CommentList: React.FC<Props> = ({ comments }) => (
+  <div className="CommentList">
+    {
+      comments.map((comment : Comment) => (
+        <CommentInfo key={comment.id} comment={comment} />
+      ))
+    }
+  </div>
```

Comment:
Use semantic tags for lists

Reply:
But in the task, we only have to split the given markup into components, and don't change class names